### PR TITLE
SERV-126 Fix enum

### PR DESCRIPTION
**Pull request by dshemin**

### CODE REVIEW

**Comment by dshemin** [comment]
Also fix SERV-127 as well.

File: testdata/jsight_0.3/req.japi.annotations/allow_enum.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	"userEnums": {
+		"@animalEnum": {
+			"annotation": "annotation",
+			"description": "",
+			"value": {
+				"key": "@animalEnum",
+				"tokenType": "object",
+				"children": [
+					{
+						"tokenType": ""
```
Comment (add2) [inline testdata/jsight_0.3/req.japi.annotations/allow_enum.json]: Here have to be
```
{
  "tokenType": "annotation",
  "scalarValue": "Pets"
}
```

File: testdata/jsight_0.3/req.japi.annotations/allow_enum.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+	"tags": {},
+	"userEnums": {
+		"@animalEnum": {
+			"annotation": "annotation",
+			"description": "",
+			"value": {
+				"key": "@animalEnum",
```
Comment (add2) [inline testdata/jsight_0.3/req.japi.annotations/allow_enum.json]: Thanks to this example, I found an error in the data schema. I'm correcting it.
However, I think we should delete the "key".

File: testdata/jsight_0.3/req.japi.annotations/allow_enum.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+				"tokenType": "object",
+				"children": [
+					{
+						"tokenType": ""
+					},
+					{
+						"key": "CAT",
```
Comment (add2) [inline testdata/jsight_0.3/req.japi.annotations/allow_enum.json]: I think we should delete all these "key".

**Review by add2 — CHANGES_REQUESTED** [review]

File: testdata/jsight_0.3/req.japi.annotations/allow_enum.json (hunk 1 above)
Comment (dshemin) [reply testdata/jsight_0.3/req.japi.annotations/allow_enum.json]: Done

File: testdata/jsight_0.3/req.japi.annotations/allow_enum.json (hunk 2 above)
Comment (dshemin) [reply testdata/jsight_0.3/req.japi.annotations/allow_enum.json]: Done

File: testdata/jsight_0.3/req.japi.annotations/allow_enum.json (hunk 3 above)
Comment (dshemin) [reply testdata/jsight_0.3/req.japi.annotations/allow_enum.json]: Done

**Review by add2 — APPROVED** [review]

**Pull request merged**